config/sync/views.view.site_subtopics.yml

```diff
@@ -0,0 +1,281 @@
+uuid: 507e80e2-fb9f-45a1-a172-062b14f2eeb7
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.site_themes
+  module:
+    - taxonomy
+    - user
+id: site_subtopics
+label: 'Site subtopics'
+module: views
+description: ''
+tag: ''
+base_table: taxonomy_term_field_data
+base_field: tid
+core: 8.x
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
+        type: perm
+        options:
+          perm: 'access content'
+      cache:
+        type: tag
+        options: {  }
+      query:
+        type: views_query
+        options:
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_comment: ''
+          query_tags: {  }
+          contextual_filters_or: 1
+      exposed_form:
```

Comment:
How does the exposed filter block fit into the use case for this view? Or is this boilerplate output that I'd misinterpreted? 😛

Reply:
There are two contextual filters for this view, both of which pull their import from the URL:
- 'parent' which checks the immediate parent of this term in the taxonomy hierarchy
- 'supplementary parent' which checks whether this term has other 'parents' in addition to the term above it in the hierarchy

'Views contextual filters OR' allows rows to be displayed by this view if they match either of the contextual filters above.

Reply:
👍